server/src/main/java/org/spine3/server/entity/FieldMasks.java

```diff
@@ -57,6 +57,30 @@ private FieldMasks() {
     }
 
     /**
+     * Creates new instance of {@code FieldMask} from given type {@link Descriptors.Descriptor descriptor} and
```

Comment:
`Creates a new instance of  {@code FieldMask} basing on the target type  {@link Descriptors.Descriptor descriptor} and field tags defined in the Protobuf message.`